keymanapp/keyman
# Android migrate folders

Previously, JS keyboards persisted in `/languages/` and accompanying fonts in `/fonts/`.
Now, keyboards and their fonts get installed into `/packages/packageID`. If `packageID` is undefined, the ID `cloud` will be used.

------
These test steps were done on Android 7.1 Nougat

## Test steps ##
1. Install previous APK and install several keyboards ("SIL IPA", "Lao 2008 Basic", "Thai Kedmanee")
2. Install Khmer Touch 1.2 via ad-hoc (legacy JSON)
3. Install the APK from this PR
4. Verified all the existing keyboards work and display correct versions
5. Uninstall some of the keyboards
6. Reinstall via cloud and ad-hoc
7. Verify keyboards work and display correct versions
